### PR TITLE
Add 21.12 LTS version as supported

### PR DESCRIPTION
**Pull request by leotada**

### CODE REVIEW

**Review by sjsadowski — APPROVED** [review]

**Review by sjsadowski — APPROVED** [review]
Change approved

**Comment by leotada** [comment]
@sjsadowski Thanks, but not yet able to merge.

**Review by ahopkins — CHANGES_REQUESTED** [review]
We need to take off 19.12 and change 20.12 to `ballot_box_with_check`

**Comment by leotada** [comment]
Done

**Review by sjsadowski — APPROVED** [review]

File: src/en/org/policies.md [hunk 1]
```diff
@@ -29,14 +29,15 @@ Sanic releases a long term support release (aka "LTS") once a year in December.
 
 | Version | LTS           | Supported               |
 | ------- | ------------- | ----------------------- |
+| 21.12   | until 2023-12 | :white_check_mark:      |
 | 21.9    |               | :white_check_mark:      |
```
Comment (ahopkins) [inline src/en/org/policies.md:33]:
```suggestion
| 21.9    |               | :x:                     |
``

**Review by ahopkins — APPROVED** [review]

**Pull request merged**